playbooks/common/openshift-master/config.yml

```diff
@@ -355,7 +355,7 @@
   hosts: oo_first_master
   tasks:
   - name: Get master service ip
-    command: "{{ openshift.common.client_binary }} get -o template svc kubernetes --template=\\{\\{.spec.clusterIP\\}\\}"
+    command: "{{ openshift.common.client_binary }} -n default --config={{ openshift.common.config_base }}/master/admin.kubeconfig get -o template svc kubernetes --template=\\{\\{.spec.clusterIP\\}\\}"
```

Comment:
I have no objections to using that admin kubeconfig here...  I just don't want to see us use it for anything that could mutate the config.